### ISSUE TITLE
Option to pause on incoming call

### ISSUE THREAD

**Issue by nikclayton** [nikclayton/android-squeezer#46]
```
There should be an option to pause the music if there's an incoming call.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2011-09-24 09:25:46

**Comment by blscorp** [comment]
Totally worthwhile feature, go for it!
Yes there are other possible interruptions, but this is a common one, and once your phone is ringing it can take a lot of rushed swiping to get to the app, silence the music and get back to to the answer screen before it goes to voicemail.

**Comment by kaaholst** [comment]
This will be available in the next version of Squeezer which is in beta now. 
You can opt in to beta [here](https://play.google.com/apps/testing/uk.org.ngo.squeezer). 
You can opt out of beta any time.

**Comment by blscorp** [comment]
That's great!  Nice going, I'll wait for the update.
Thanks for the alert.

**Comment by kaaholst** [comment]
This is available in Squeezer version 2.2.2, so closing this issue.
